src/js/shared.js

```diff
@@ -623,11 +625,14 @@ function cardButtons (t) {
                                 memberRanges.forEach((range) => {
                                     const start = new Date(range.item.start * 1000);
                                     const end = new Date(range.item.end * 1000);
+									const rangeOnTheSameDay = start.toDateString() === end.toDateString();
+									const rangeLengthInSeconds = range.item.end - range.item.start;
+
                                     const _rangeIndex = range.rangeIndex;
                                     const _range = range;
 
                                     items.push({
-                                        text: formatDate(start) + ' - ' + formatDate(end),
+                                        text: formatDate(start) + ' - ' + formatDate(end, rangeOnTheSameDay) + ' <span class="text-muted">('+ formatTime(rangeLengthInSeconds, false) +')</span>',
```

Comment:
@jonez1 Unfortunately you cannot use HTML for the default components in Trello. I'm debating with myself whether or not to completely overhaul the project from scratch with custom views for these lists. So that we can properly style it. But for now we cannot use HTML. You can see how it looks here: https://prnt.sc/xxnxsv - can you change it to just be without HTML for the time being?

Reply:
@danniehansen Fixed in https://github.com/danniehansen/activity-timer/pull/30/commits/3237ec9a46a1aaa816a4de292cf97e7cadbb720c